data/operators/amenity/charging_station.json

```diff
@@ -2250,6 +2261,18 @@
         "operator:wikipedia": "en:Meridian Energy"
       }
     },
+    {
+      "displayName": "MER",
```

Comment:
There is already [Mer](https://nsi.guide/index.html?t=brands&k=amenity&v=charging_station&tt=mer#mer-b20835). Are they the same?

Reply:
I did spot that soon after I did the pull request. You're right. There were duplicates in the wikidata, which I've now flagged.